src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateSplit = date.split(fromFormat[3]);
+  const newDate = [];
+
+
+  for (let i = 0; toFormat.length - 1 > i; i++) {
+    const format = toFormat[i];
+    console.log(format)
```

Comment:
Remove the `console.log` statements before finalizing the function. Debugging statements should not be present in production code.